esrally/client.py

```diff
@@ -12,7 +12,12 @@ class EsClientFactory:
     Abstracts how the Elasticsearch client is created. Intended for testing.
     """
     def __init__(self, hosts, client_options):
-        logger.info("Creating ES client connected to %s with options [%s]" % (hosts, client_options))
+        masked_client_options = dict(client_options)
+        if masked_client_options.has_key("basic_auth_password"):
```

Comment:
Can you please change this to `"basic_auth_password" in masked_client_options`?

Reply:
Yes. I learned it's more pyhtonic and Pyhton 3 compatible: https://stackoverflow.com/questions/1323410/has-key-or-in